.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     steps:
       - checkout
       - run:
-          command: pip install --user tox coverage coveralls
```

Comment:
The latest version of coverage is incompatible with the latest version of coveralls. Let covaralls install preferred version instead.